src/Ch9/Ch9.Shared/MainPage.xaml

```diff
@@ -10,9 +9,8 @@
 	  xmlns:lottie="using:Microsoft.Toolkit.Uwp.UI.Lottie"
 	  xmlns:winui="using:Microsoft.UI.Xaml.Controls"
 	  xmlns:android="http://Uno.UI/android"
-	  xmlns:xamarin="http://Uno.UI/xamarin"
 	  xmlns:triggers="using:WindowsStateTriggers"
-	  mc:Ignorable="android xamarin">
+	  mc:Ignorable="android">
```

Comment:
Changes on this file will be remobed from this PR once previous one merged and this one rebased.